commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -418,7 +418,34 @@ public CommerceOrder checkoutCommerceOrder(
 			commerceOrder.setShippingAddressId(shippingAddressId);
 		}
 
-		return commerceOrderPersistence.update(commerceOrder);
+		commerceOrderPersistence.update(commerceOrder);
+
+		// Low stock action
+
+		for (CommerceOrderItem commerceOrderItem :
+				commerceOrder.getCommerceOrderItems()) {
+
+			TransactionCommitCallbackUtil.registerCallback(
+				new Callable<Void>() {
+
+					@Override
+					public Void call() throws Exception {
+						Message message = new Message();
+
+						message.put(
+							"cpInstanceId",
+							commerceOrderItem.getCPInstanceId());
+
+						MessageBusUtil.sendMessage(
```

Comment:
@Commando18 This is a check that make sense to put in the consume quantity method of the inventory engine, and I would like to get an integration test with this fix that cover it